crowdin_api/api_resources/screenshots/resource.py

```diff
@@ -47,7 +49,8 @@ def list_screenshots(
         return self._get_entire_data(
             method="get",
             path=self.get_screenshots_path(projectId=projectId),
-            params=self.get_page_params(page=page, offset=offset, limit=limit),
+            params=self.get_page_params(
+                page=page, offset=offset, limit=limit, labelIds=labelIds, excludeLabelIds=excludeLabelIds),
```

Comment:
this is not a page params, it's query params of the request

example - https://github.com/crowdin/crowdin-api-client-python/blob/main/crowdin_api/api_resources/projects/resource.py#L64